packages/lab/stories/stepped-tracker/stepped-tracker.qa.stories.tsx

```diff
@@ -21,7 +21,7 @@ export const Basic: StoryFn<QAContainerProps> = (props) => {
           marginBottom: 50,
         }}
       >
-        <SteppedTracker activeStep={0}>
+        <SteppedTracker orientation="vertical" activeStep={0}>
```

Comment:
i think these ones within `Basic` example are meant to be `orientation="horizontal"` as there's a `Vertical` example below